remote-sources/kubernetes/src/object-types.ts

```diff
@@ -0,0 +1,31 @@
+export type GqlSchemaConfig = string;
+
+export type GqlEndpointConfig = {
+    host: string,
+    headers: {
+        name: string,
+        value: string,
+    }[],
+    auth: {
+        type: string,
+        authority: string,
+        scope: string,
+    }
+};
+
+
+export type GqlAuthProviderConfig = {
+    type: string,
+    authority: string,
+    client_id: string,
```

Comment:
I also would change here the client_id and client_secret to `clientId` and `clientSecret`

Reply:
Fixed